### DISCUSSION TITLE
How use geoip-detect with WPForms form?

### DISCUSSION THREAD

**Discussion by GitHubKitten** [yellowtree/geoip-detect#251] [Q&A]
I am wondering wow use geoip-detect with WPForms form (Lite).
I've been having some issues with CF7 so I switched to WPForms.
Now, I want to use shortcodes from geoip-detect or add some functions to functions.php to allow me to get and include IP/country in the notification email from the WPForms submitted form.

I would really appreciate if there is a way to do this.

**Comment by benjaminpick** [comment]
Good idea, actually this would be pretty easy to implement. I have created an issue for this, however I cannot promise any timeline for this. It's more probable if somebody is sending me a pull request for this.
https://github.com/yellowtree/geoip-detect/issues/252

**Reply by benjaminpick** [reply]
I have released 5.4.0-beta1 with basic support now. The tags are being replaced but it doesn't look nice (depending on the E-Mail template you choose ... not sure if I can fix this.)

Can you test it please?
https://github.com/yellowtree/geoip-detect/wiki/Beta-Testing
